src/components/viewLayout/viewLayout.tsx

```diff
@@ -106,17 +107,11 @@ const AppLayout: React.FunctionComponent<IAppLayout> = ({ children }) => {
     </SkipToContent>
   );
 
-  const isLogin = location.pathname === '/login';
   return (
-    <Page
-      mainContainerId={pageId}
-      header={!isLogin && Header}
-      sidebar={!isLogin && sidebarOpen && Sidebar}
-      skipToContent={PageSkipToContent}
-    >
+    <Page mainContainerId={pageId} header={Header} sidebar={sidebarOpen && Sidebar} skipToContent={PageSkipToContent}>
       {children}
     </Page>
   );
 };
 
-export default AppLayout;
+export { AppLayout as default, AppLayout, type AppLayoutProps };
```

Comment:
required: since you're exporting props confirm the other components you've updated are consistent